src/SFA.DAS.LevyTransferMatching.Functions/Events/ApplicationApprovedEventHandlerForReceiverNotification.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using RestEase;
+using SFA.DAS.Encoding;
+using SFA.DAS.LevyTransferMatching.Functions.Api;
+using SFA.DAS.LevyTransferMatching.Infrastructure;
+using SFA.DAS.LevyTransferMatching.Infrastructure.Configuration;
+using SFA.DAS.LevyTransferMatching.Messages.Events;
+using SFA.DAS.NServiceBus.AzureFunction.Attributes;
+
+namespace SFA.DAS.LevyTransferMatching.Functions.Events
+{
+    public class ApplicationApprovedEventHandlerForReceiverNotification
+    {
+        private readonly ILevyTransferMatchingApi __levyTransferMatchingApi;
```

Comment:
You may have accidentally given this a double underscore prefix, which means it follows no convention.

Reply:
Extra underscore removed now.